LucidGloves/src/ControllerDriver.cpp

```diff
@@ -87,10 +87,11 @@ void ControllerDriver::StartDevice() {
 
 			try {
 				//Compute each finger transform
-				int currentBoneIndex = 2;
-				for (int i = 0; i < datas.flexion.size(); i++) {
-					ComputeBoneTransform(m_handTransforms, datas.flexion[i], currentBoneIndex, IsRightHand());
-					currentBoneIndex = currentBoneIndex == 2 ? currentBoneIndex += 4 : currentBoneIndex += 5;
+				for (int i = 0; i < NUM_BONES; i++) {
```

Comment:
maybe this loop should be in `Bones.cpp`?

Reply:
You mean like in a ComputeEntireHand function? Or ComputeEntireFinger?

Reply:
In fact, I think it's probably fine to have this loop there, but let's continue thinking about it.